fs/source/source.go

```diff
@@ -181,13 +181,13 @@ func AppendDefaultLabelsHandlerWrapper(indexDigest string, wrapper func(images.H
 							c.Annotations = make(map[string]string)
 						}
 
-						c.Annotations[TargetSizeLabel] = fmt.Sprintf("%d", c.Size)
+						c.Annotations[TargetSizeLabel] = strconv.FormatInt(c.Size, 10)
 						c.Annotations[TargetSociIndexDigestLabel] = indexDigest
 
 						var layerSizes string
 						for _, l := range children[i:] {
 							if images.IsLayerType(l.MediaType) {
-								ls := fmt.Sprintf("%d,", l.Size)
+								ls := strconv.FormatInt(l.Size, 10)
```

Comment:
this is missing a trailing comma in the resulting string

Reply:
Oh nice catch. Let me revert it here.

Reply:
`FormatInt` + `+` might still be faster, but no need to benchmark or make that change here.

Reply:
Alternatively could have done something like this:

```go
ls := strconv.FormatInt(l.Size, 10) + ","
```

which from profiling is still faster even with string concatenation; however, was hoping to avoid such micro optimizations and only focus on intuitive swaps. It was more of a fun project, not really expecting a noticeable performance diff.